pysap/base/loaders/tif.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+##########################################################################
+# pySAP - Copyright (C) CEA, 2017 - 2018
+# Distributed under the terms of the CeCILL-B license, as published by
+# the CEA-CNRS-INRIA. Refer to the LICENSE file or to
+# http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html
+# for details.
+##########################################################################
+
+# Package import
+from .loader_base import LoaderBase
+from pysap.base.image import Image
+
+# Third party import
+from skimage.io import imread, imsave
+
+
+class TIF(LoaderBase):
+    """ Define the '.tif' file loader.
+    """
+    allowed_extensions = [".tif"]
+
+    def load(self, path, as_gray=False):
+        """ A method that load the data in TIF format.
+
+        Parameters
+        ----------
+        path: str
+            the path to the data to be loaded.
+        meta_field: str, default 'metadata'
```

Comment:
`meta_field` is not one of the method arguments and `as_gray` is missing.

Reply:
I don't understand I can no longer edit the files to fix the above mentioned issue. That's really messy!